### ISSUE TITLE
Document changes from 0.17 to 1.0

### ISSUE THREAD

**Issue by nMustaki** [jfinkels/flask-restless#535]
Hi,

thanks for your project !
Could you take the time to document a migration path from 0.17 to 1.0 ? I have no idea how long and how complex it would be to update and that's a bit scarry !
I will do it for me anyways, but your input would be much appreciated :-)

Thanks again,
Nathan

**Comment by jfinkels** [comment]
Yes, this is absolutely needed, and is something I should have done when I first started making changes that will appear in 1.0.0. However, since I'm not a professional, I failed to do so. I will do this before I make the final release.

**Comment by nMustaki** [comment]
Don't blame yourself, flask-restless is amazing :-)
I'll try to update to 1.0beta soon and I'll document the process to share
it here in case it is of any help.


--
/*
** "What do you despise? By this you are truly known."
**	from Manual of Muad'Dib by the Princess Irulan.
*/

**Comment by jfinkels** [comment]
@seanharr11 has provided a good chunk of information on migrating from the 0.17.0 to 1.0.0: http://thelaziestprogrammer.com/sharrington/web-development/upgrading-to-flask-restless-v1.0.0-part-1

(I will likely use that post as a starting point for a migration guide in the documentation.)
